docs/WindowsBuild.md

```diff
@@ -36,8 +42,17 @@ git clone -c core.autocrlf=input https://github.com/apple/swift-package-manager
 git clone https://github.com/compnerd/windows-swift windows-swift
 ```
 
-## Acquire ICU, SQLite3, curl, libxml2, zlib
-1. Go to https://dev.azure.com/compnerd/windows-swift and scroll down to "Dependencies" where you'll see bots (hopefully green) for icu, SQLite, curl, and libxml2. Download each of the zip files and copy their contents into S:/Library. The directory structure should resemble:
+## Acquire ICU, SQLite3, curl, libxml2 and zlib
+
+Go to [compnerd's windows-swift azure page](https://dev.azure.com/compnerd/windows-swift/_build) and open [Pipelines](https://dev.azure.com/compnerd/windows-swift/_build) where you'll see bots (hopefully green) for:
+
+- [ICU](https://dev.azure.com/compnerd/windows-swift/_build?definitionId=9)
+- [SQLite](https://dev.azure.com/compnerd/windows-swift/_build?definitionId=12&_a=summary)
+- [curl](https://dev.azure.com/compnerd/windows-swift/_build?definitionId=11&_a=summary)
+- [libxml2](https://dev.azure.com/compnerd/windows-swift/_build?definitionId=10&_a=summary)
+- [zlib](https://dev.azure.com/compnerd/windows-swift/_build?definitionId=16&_a=summary)
```

Comment:
Good idea with the links.  However, I think that we can actually do this much better now.  I've been playing around with more automation and something like this works really well:

```
pip install azure
C:\Python27\python.exe windows-swift\utilities\windows-swift.py --build 10 --build 11 --build 12 --build 16 --download-latest-artifacts
```

Reply:
Thanks :)
I agree that the command looks great. We can keep the link for information purpose. What do you think ?

Reply:
Sure